pilot/cmd/pilot-agent/config.go

```diff
@@ -30,6 +32,29 @@ import (
 	"istio.io/pkg/log"
 )
 
+// getTLSCerts returns all file based certificates from mesh config
+// TODO(https://github.com/istio/istio/issues/21834) serve over SDS instead of files
+func getTLSCerts(pc meshconfig.ProxyConfig) []string {
+	certs := []string{}
+	appendTLSCerts := func(rs *meshconfig.RemoteService) {
```

Comment:
More comments please. 

What are they used for ? How are they distributed to sidecars ? Do we still need them - i.e. stable/beta features we support ? Can we use a generic API ( like Sidecar with Shriram's new TLS support ) to generalize the load of certs we need for various purpose ? 

At least as TODO - I understand you are just cleaning up.

Reply:
added some more context. they are file mounted though